addons/mail/__manifest__.py

```diff
@@ -2,7 +2,7 @@
 
 {
     'name': 'Discuss',
-    'version': '1.7',
+    'version': '1.8',
```

Comment:
Don't increase when you have no model modification. Version increase in SM apps is done only when we really modify models; here it is only data cleaning in DB.